JSDemos/Demos/Scheduler/RecurringAppointments/description.md

```diff
@@ -1 +1,19 @@
-This example demonstrates how to&nbsp;define recurring appointments. You can specify the appointment repetition interval programmatically or&nbsp;in&nbsp;the UI. In&nbsp;the former case, the appointment object should contain the **recurrenceRule** field. In&nbsp;the latter case, toggle the **Repeat** switcher on&nbsp;the appointment details form to&nbsp;mark the appointment as&nbsp;recurring and specify the required fields.
\ No newline at end of file
+A recurring appointment repeats at a specified interval. You can create such an appointment in the UI or programmatically.
+
+To create a recurring appointment programmatically, do the following:
+
+1. Locate or create an appointment object in the [dataSource](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#dataSource). 
+
+2. Specify the [recurrenceRule](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceRule) property to configure the appointment frequency. Set this property according to the <a href="http://tools.ietf.org/html/rfc2445#section-4.3.10" target="_blank">iCalendar RFC 2445</a> specification. Note that an appointment series corresponds to a single object in the data source.
+
+1. Optionally, use the [recurrenceException](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceException) property to specify the start date and time of appointments to exclude from the series.
+
+To create a recurring appointment in the UI, users should do the following:
+
+1. Create or open an appointment.
+
+2. Toggle the *"Repeat"* switcher on the appointment details form.
+
+3. Use the form's fields to set the recurrence rules.
```

Comment:
A list is a bit of an overshoot here. However, we do need to outline the mechanics of editing recurrence rules from the ui.

```suggestion
Users also can create recurrent appointments. The appointment details form contains a Repeat switcher. When it is on, the form displays an additional set of fields that allow users to specify recurrence rules. Double-click an appointment and select Edit Series to view the appointment details form with the recurrence fields.
```